pkg/costmodel/allocation_helpers.go

```diff
@@ -18,7 +18,10 @@ import (
 
 // This is a bit of a hack to work around garbage data from cadvisor
 // Ideally you cap each pod to the max CPU on its node, but that involves a bit more complexity, as it it would need to be done when allocations joins with asset data.
-const MAX_CPU_CAP = 512
+const CPU_SANITY_LIMIT = 512
+
+// Sanity Limit for PV usage, set to 10 PB, in bytes for now
+const PV_USAGE_SANITY_LIMIT = 10737418240
```

Comment:
Very small nit -- can we add `_BYTES` to the end of the name here? Trying to get the whole team on board for adding units to names when we can, to avoid ambiguity.

Reply:
Absolutely, that's a sensible semantic change

Reply:
Thank you!